src/common/packager.ts

```diff
@@ -220,52 +223,52 @@ export class Packager {
                         if (!silent) {
                             this.logger.warning(ErrorHelper.getWarning(localize("PackagerIsStillRunning", "Packager is still running. If the packager was started outside VS Code, please quit the packager process using the task manager.")));
                         }
-                        return Q.resolve<void>(void 0);
+                        return Promise.resolve();
                     }
                     return this.killPackagerProcess();
                 } else {
                     if (!silent) {
                         this.logger.warning(ErrorHelper.getWarning(localize("PackagerIsNotRunning", "Packager is not running")));
                     }
-                    return Q.resolve<void>(void 0);
+                    return Promise.resolve();
```

Comment:
Why don't we need to pass the `void 0` value to the `resolve` function  here and in some other places in the code?

Reply:
`void 0` returns `undefined` the same way as no arguments. It was the requirement for Q but it is not for native promises

Reply:
I just saw that in some places of the refactored code `void 0` was removed while in others it was left without changes.

Reply:
Maybe I missed something, I'll take a look in future PRs, thanks.